abstract-media-service/src/main/java/org/arpha/service/MediaService.java

```diff
@@ -0,0 +1,18 @@
+package org.arpha.service;
+
+import org.arpha.dto.media.request.FileUploadRequest;
+import org.arpha.dto.media.response.FileResponse;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.multipart.MultipartFile;
+
+public interface MediaService {
+
+    FileResponse upload(MultipartFile multipartFile, FileUploadRequest fileUploadRequest);
```

Comment:
We should not work directly with multipartFiles as it will critically affect the system's performance. Please consider creating sas link with write access to specific (not yet) created blob and leave uploading itself to the client